Fixed image resize bug

wpichler opened the pull request:
When used with product_template for example - it will overwrite the original image with the smaller images which get automatically generated by product_template. It does not make sense here to store the smaller images - because we do resize the images on the fly !

StefanRijnhart committed:
[FIX] Request id no longer exists after concurrency rollback

wpichler committed:
Removed setting of resized images, this caused the original image getting overwritten my smaller version

wpichler committed:
Removed setting of resized images, this caused the original image getting overwritten my smaller version

wpichler committed:
Merge branch '9.0' of https://github.com/Callino/server-tools into 9.0

pedrobaeza commented:
You are putting product as example, but does it fit in all models? This is one is the generic one, and the small and medium image is for each sub-image I think. Please check this and also the Travis status.

hparfr committed:
Keychain: account manager for external systems (#644)

* Add keychain module

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
OCA Transbot updated translations from Transifex

Commit:
Improve readme and fix dep

Readme: (in example) prefix function with _
Dep: add a depency to base_setup in order to use xml id "base.menu_config"

hparfr committed:
Update __openerp__.py

Remove explicit dependency to base.

legalsylvain committed:
Merge pull request #733 from akretion/keychain-improve_readme_and_dep

Improve readme and fix dep

oca-transbot committed:
OCA Transbot updated translations from Transifex

pedrobaeza committed:
Merge pull request #745 from StefanRijnhart/9.0-auditlog_concurrency_rollback

[FIX] Request id no longer exists after concurrency rollback

oca-transbot committed:
OCA Transbot updated translations from Transifex

obulkin committed:
[9.0][ADD] auth_totp: MFA Support (#687)

* [ADD] auth_totp: MFA Support
* Add relevant fields and methods to the res.users model
* Overload check_credentials in res.users to allow for logins using an MFA
login token rather than a password
* Add the res.users.authenticator and res.users.device models, along with
appropriate ACLs and record rules
* Add the res.users.authenticator.create wizard model and an associated view
to facilitate creation of res.users.authenticator records
* Extend base.view_users_form_simple_modif with fields needed to manage the
new functionality
* Add an AuthTotp controller that inherits from Home in the web module and an
associated view to introduce MFA logic to the login process
* Add several new exception classes that inherit from AccessDenied

* PR commit

* PR commit

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
OCA Transbot updated translations from Transifex

oca-transbot committed:
OCA Transbot updated translations from Transifex

sbidoul committed:
[UPD] fix website url in manifest

yajo committed:
Bump version

See https://github.com/OCA/server-tools/pull/728#issuecomment-279772432

OCA-git-bot committed:
[UPD] addons table in README.md

wpichler committed:
Merge branch '9.0' of https://github.com/Callino/server-tools into 9.0

pedrobaeza commented:
This branch is incorrect with lots of commits. Please create a separate one and then commit there your change attending my comments. I close meanwhile.
